frontend/src/components/enroll/enroll.jsx

```diff
@@ -16,29 +17,32 @@ class Enroll extends Component {
       }
     render() {
       return (
-        <div>
+        <div class="main-content">
             <Markdown source={this.state.text} />
             
-            <div id="contact-form-900">
+            <div class='jumbotron'>
                 <form action="/" method="post" class="contact-form commentsblock">
 
                     <div class="grunion-field-wrap grunion-field-name-wrap">
-                        <label for="g900-name" class="grunion-field-label name">Name<span>(required)</span></label>
-                        <input type="text" name="g900-name" id="g900-name" value="" class="name" required="" aria-required="true" />
+                        <label for="g900-name" class='col-md-2'>Name<span> (required)</span></label>
```

Comment:
I know it's getting repetitive, but go ahead and strip out the old CSS & names here too. I'm v tired of g-whatever.

Reply:
fixed